mwclient/client.py

```diff
@@ -421,7 +499,19 @@ def raw_call(self, script, data, files=None, retry_on_error=True, http_method='P
                 sleeper.sleep()
 
     def raw_api(self, action, http_method='POST', *args, **kwargs):
-        """Send a call to the API."""
+        """Send a call to the API.
+
+        Args:
+            action (str): The MediaWiki API action to perform.
+            http_method (str): The HTTP method to use in the request.
```

Comment:
as well as `action` and `http_method`, this effectively takes `retry_on_error` and any args that `_query_string` accepts (since it passes `*args` and `**kwargs` to `_query_string`).

Reply:
What do you think about just adding an explicit `retry_on_error=True` to the function signature here? This should be a non-breaking change, as it merely makes a kwarg explicit, and makes the intention obvious to users.

Reply:
yeah, that seems pretty reasonable to me.

Reply:
Alright, I've added a new commit to this PR making that change. Thanks :)